nbextensions/bulkImportCell/bulkImportCellStates.js

```diff
@@ -154,21 +142,22 @@ define([], () => {
         appError: {
             ui: {
                 tab: tabState(
-                    ['viewConfigure', 'info', 'jobStatus', 'results', 'error'],
-                    ['viewConfigure', 'info', 'jobStatus', 'results', 'error']
+                    ['viewConfigure', 'info', 'jobStatus', 'error'],
```

Comment:
Hide the results tab, replace it with the error tab. This state can only be reached at present when the request to run a batch job returns an error. We may be able to merge it into the general error, below.